tests/IntegrationTest.php

```diff
@@ -73,4 +74,53 @@ public function gettingEncryptedStuffFromGoogleShouldWork()
         $this->assertTrue($connected);
         $this->assertRegExp('#^HTTP/1\.0#', $response);
     }
+
+    /** @test */
+    public function testSelfSignedRejectsIfVerificationIsEnabled()
```

Comment:
Can you give some error output for the failing test?

Unfortunately I do not have a Mac do test against, so how else could I assist you tracking this down?

As an alternative, how about skipping this test on Mac for now (similar to how we skip certain tests for HHVM)?